packages/google_sign_in/google_sign_in_web/example/pubspec.yaml

```diff
@@ -2,8 +2,8 @@ name: google_sign_in_web_integration_tests
 publish_to: none
 
 environment:
-  sdk: ^3.2.0
-  flutter: ^3.16.0
+  sdk: ">=3.2.0 <4.0.0"
```

Comment:
`^` should be fine here! Weird, I know...

Reply:
Haha yeah, I noticed that this is a flutter-only thing, however I saw other entries of the sdk like that one so I decided to make them the same to make it easier to find-replace in the future? maybe?

Reply:
🤷 no biggy!